buildout.cfg

```diff
@@ -32,6 +33,17 @@ working-directory = ${buildout:directory}/docs
 defaults = ['--auto-color']
 eggs = crash [test]
 
+[linter]
+recipe = zc.recipe.egg:script
+eggs = flake8
+relative-paths=true
+scripts = flake8=linter
+initialization =
+    sys.argv.extend(['--ignore', 'E501,C901',
```

Comment:
can we add these options in a `setup.cfg` file instead so that they're also picked up by linter editor-plugins?

Reply:
ah, did not know that this was possible. sure thing, will change it.